tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -500,8 +500,9 @@ def param_value(self, verbose, name, ptype, value, textfile, binfile, command):
                 print("... executing command param [%s]" % command)
             import subprocess, shlex #shlex rocks
             try:
-                os_posix = os.name == "posix"
-                p = subprocess.Popen(shlex.split(command, posix=os_posix), stdout=subprocess.PIPE)
+                if os.name == "posix":
```

Comment:
>os.name == "posix": [](start = 19, length = 19)

The same check posfix=os_posfox was added to multiple locations, do we need to revise those changes, too?